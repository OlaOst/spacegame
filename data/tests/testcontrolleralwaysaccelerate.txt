test.control = AlwaysAccelerate
test.thrustForce = 1.0
test.mass = 1.0