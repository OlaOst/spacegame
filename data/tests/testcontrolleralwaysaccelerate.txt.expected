test.control = AlwaysAccelerate
test.thrustForce = 1.0
test.force = [0, 1]